client/v2/common/models/pending_transaction_response.go

```diff
@@ -26,18 +26,18 @@ type PendingTransactionResponse struct {
 	// ConfirmedRound the round where this transaction was confirmed, if present.
 	ConfirmedRound uint64 `json:"confirmed-round,omitempty"`
 
-	// GlobalStateDelta (gd) Global state key/value changes for the application being
+	// GlobalStateDelta global state key/value changes for the application being
 	// executed by this transaction.
 	GlobalStateDelta []EvalDeltaKeyValue `json:"global-state-delta,omitempty"`
 
 	// InnerTxns inner transactions produced by application execution.
 	InnerTxns []PendingTransactionResponse `json:"inner-txns,omitempty"`
 
-	// LocalStateDelta (ld) Local state key/value changes for the application being
-	// executed by this transaction.
+	// LocalStateDelta local state key/value changes for the application being executed
+	// by this transaction.
 	LocalStateDelta []AccountStateDelta `json:"local-state-delta,omitempty"`
 
-	// Logs (lg) Logs for the application being executed by this transaction.
+	// Logs logs for the application being executed by this transaction.
 	Logs [][]byte `json:"logs,omitempty"`
```

Comment:
Did the new node status fields fail to generate?

https://github.com/algorand/java-algorand-sdk/pull/555/files#diff-c8678c6feccbb68be3a4c002342c378832eaf4a03b2db6ff92747f13ef3d3874

Reply:
They were generated but I didn't add them here - I can do that in the next commit

Reply:
This is for the simulate endpoint

It looks like node status fields were already merged: https://github.com/algorand/go-algorand-sdk/blob/develop/client/v2/common/models/node_status_response.go

Reply:
oops! ty for the correction.